src/status_im/models/browser-history.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im.models.browser-history
+  (:require [re-frame.core :as re-frame]))
+
+(defn dont-store-history-on-nav-change? [db]
+  (get-in db [:browser/options :dont-store-history-on-nav-change?]))
+
+(defn dont-store-history-on-nav-change []
```

Comment:
maybe adding a `!` to the name? otherwise they look very similar the one with `?` & without (same for the method below, for consistency).

Reply:
Done!